### ISSUE TITLE
Version 2.3.0.32 - Could not load file or assembly System.Data.SQLite, Version=1.0.66.0

### ISSUE THREAD

**Issue by avorobjovs** [UweKeim/ZetaResourceEditor#45]
Hi,

After updating the app to version 2.3.0.32 I cannot export resources to an Excel file anymore. I always have the error message:

```
System.BadImageFormatException

-----------------

Could not load file or assembly '![ZetaResourceEditor_SystemDataSQLite](https://user-images.githubusercontent.com/7294773/140056310-11dc6e95-dbc6-4d13-b4c4-80f27ffc13c7.png), Version=1.0.66.0, Culture=neutral, PublicKeyToken=db937bc2d44ff139' or one of its dependencies. An attempt was made to load a program with an incorrect format.

-----------------

at ZetaResourceEditor.RuntimeBusinessLogic.Snapshots.SnapshotControllerBase.checkTablesCreated()
   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.doProcess(PreparedInformation preparedInformation, BackgroundWorker bw, Int32 fileGroupsLength, ICollection`1 savedFiles) in c:\P\ZetaResourceEditor\Source\RuntimeBusinessLogic\ExportImportExcel\Export\ExcelExportController.cs:line 321
   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.Process(BackgroundWorker bw) in c:\P\ZetaResourceEditor\Source\RuntimeBusinessLogic\ExportImportExcel\Export\ExcelExportController.cs:line 300
   at System.ComponentModel.BackgroundWorker.OnDoWork(DoWorkEventArgs e)
   at System.ComponentModel.BackgroundWorker.WorkerThreadStart(Object argument)

BadImageFormatException
{
	Message="Could not load file or assembly \\'System.Data.SQLite, Version=1.0.66.0, Culture=neutral, PublicKeyToken=db937bc2d44ff139\\' or one of its dependencies. An attempt was made to load a program with an incorrect format.",
	FileName="System.Data.SQLite, Version=1.0.66.0, Culture=neutral, PublicKeyToken=db937bc2d44ff139",
	FusionLog="WRN: Assembly binding logging is turned OFF.\\r\\nTo enable assembly bind failure logging, set the registry value [HKLM\\Software\\Microsoft\\Fusion!EnableLog] (DWORD) to 1.\\r\\nNote: There is some performance penalty associated with assembly bind failure logging.\\r\\nTo turn this feature off, remove the registry value [HKLM\\Software\\Microsoft\\Fusion!EnableLog].\\r\\n",
	Data=
	[
	]
,
	InnerException=null,
	TargetSite=RuntimeMethodInfo: checkTablesCreated,
	StackTrace="   at ZetaResourceEditor.RuntimeBusinessLogic.Snapshots.SnapshotControllerBase.checkTablesCreated()\\r\\n   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.doProcess(PreparedInformation preparedInformation, BackgroundWorker bw, Int32 fileGroupsLength, ICollection`1 savedFiles) in c:\\P\\ZetaResourceEditor\\Source\\RuntimeBusinessLogic\\ExportImportExcel\\Export\\ExcelExportController.cs:line 321\\r\\n   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.Process(BackgroundWorker bw) in c:\\P\\ZetaResourceEditor\\Source\\RuntimeBusinessLogic\\ExportImportExcel\\Export\\ExcelExportController.cs:line 300\\r\\n   at System.ComponentModel.BackgroundWorker.OnDoWork(DoWorkEventArgs e)\\r\\n   at System.ComponentModel.BackgroundWorker.WorkerThreadStart(Object argument)",
	HelpLink=null,
	Source="ZetaResourceEditor.RuntimeBusinessLogic",
	HResult=-2147024885
}
```

But I found the file System.Data.SQLite.dll in the "C:\Program Files (x86)\Zeta Resource Editor" folder and its version is 1.0.66.0.

![ZetaResourceEditor_SystemDataSQLite](https://user-images.githubusercontent.com/7294773/140056438-4e804617-e8fc-45f6-b8da-5872c6efa6b5.png)

**Comment by UweKeim** [comment]
Thanks for reporting, I'll take a look.

**Comment by UweKeim** [comment]
Just released Version `2.3.0.33` that should fix this issue.

[📎 Direct download](https://zeta.li/zre-download-setup)

**Comment by avorobjovs** [comment]
This 2.3.0.33 version works fine. Thank you!
You can close the issue.
